pkg/tfvars/vsphere/vsphere.go

```diff
@@ -76,3 +94,39 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 
 	return json.MarshalIndent(cfg, "", "  ")
 }
+
+func getNetworkMoID(vcenter, username, password, datacenter, cluster, network string) (string, error) {
+	client, _, err := vsphere.CreateVSphereClients(context.TODO(), vcenter, username, password)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to setup vSphere client")
+	}
+
+	finder := find.NewFinder(client)
+	path := fmt.Sprintf("/%s/host/*", datacenter)
```

Comment:
Wondering since we know the cluster if we just use it in the path?
And then remove the branch starting on line 113

Reply:
I think you are right. I believe we can get rid of the loop as well if we use `finder.ClusterComputeResource` instead of `List`.